snapcraft/integrations/travis.py

```diff
@@ -53,7 +53,17 @@
         snapcraft && snapcraft push *.snap --release edge"
       on:
         branch: master
+
+The dedicated credentials will be functional for exactly 1 year or until
+a password change on the related Ubuntu One SSO account. In order to
+refresh the project credentials, please run the following command::
+
+    $ snapcraft enable-ci travis --refresh
```

Comment:
Won't macaroon be refreshed on every `push` and/or `release` as well? Maybe the 1 year since last use?